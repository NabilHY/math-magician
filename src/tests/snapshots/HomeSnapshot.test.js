import renderer from 'react-test-renderer';
import Home from '../../component/home';

describe('home component renders correctely', () => {
  it('main page matches snapshot', () => {
    const tree = renderer.create(<Home />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
