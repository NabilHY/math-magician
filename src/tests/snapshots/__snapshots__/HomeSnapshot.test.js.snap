// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`home component renders correctely main page matches snapshot 1`] = `
<section>
  <h2>
    Welcome to The Math Magicien
  </h2>
  <p>
     Math magicien was built using React library
  </p>
  <p>
    Feel free to navigate throught the section on this website.
  </p>
  <p>
    Click on the Calculator above to open an application that does simple Math operations for you
  </p>
  <p>
    Click on the quote to generate a random math related quote by a famous mathmaticien
  </p>
</section>
`;
